src/main/resources/l10n/JabRef_da.properties

```diff
@@ -1110,7 +1110,8 @@ Main\ layout\ file=Hoved-layoutfil
 
 Always\ add\ letter\ (a,\ b,\ ...)\ to\ generated\ keys=Tilføj altid bogstav (a, b, ...) til genererede nøgler
 Default\ pattern=Standardmønster
-Display\ quantity\ of\ items\ in\ group=
+Display\ count\ of\ items\ in\ group=
```

Comment:
You only have to add this to en properties file

Reply:
Oh, sorry, sorry, sorry. Is it OK like this in "en" file?
_Display\ count\ of\ items\ in\ group=Display count of items in group_

Reply:
it's correct now! (you can check l10n correctness with the LocalizationConsistencyTest)

Reply:
Thanks.